utils/cclyrics.js

```diff
@@ -0,0 +1,39 @@
+const cheerio = require('cheerio');
+
+const { asyncRequest } = require('./utils.js');
+
+const baseUrl = 'https://colorcodedlyrics.com/?s=';
+
+function _cleanSearchString(str) {
+    return str.toLowerCase()
+        .replace(/\b\sby\s\b|\b\s\-\s\b/g, ' ')
+        .replace(/\s\s+/g, ' ')
+        .trim();
+};
+
+function _buildCCLyricsSearchUrl(str) {
+    const encodedString = encodeURIComponent(str);
+    return `${baseUrl}${encodedString}`;
+};
+
+async function getCCLyricsUrl(searchString) {
+    const cleanedSearchString = _cleanSearchString(searchString);
+    const ccLyricsSearchUrl = _buildCCLyricsSearchUrl(cleanedSearchString);
+    const ccLyricsResultsPage = await asyncRequest(ccLyricsSearchUrl);
+    const $ = cheerio.load(ccLyricsResultsPage);
```

Comment:
wow new JQuery??

Reply:
Yeah, not too familiar with cheerio but we should probably just let the framework do its thing